### PR TITLE
DevRel Ekaansh - Improvements for Web & React Native 

### PR DESCRIPTION
**Pull request by EkaanshArora**
Moving #306 to an internal branch as requested.

### CODE REVIEW

File: shared/video-sdk/get-started/get-started-uikit/project-implementation/web.mdx [hunk 1]
```diff
@@ -83,27 +98,27 @@ export default App;
 
 ### Display the user interface
 
-If `videoCall` is `true`, show the `AgoraUIKit` interface. If it is `false` show the _Join_ button. To do this, add the provided code sample after the `callbacks` declaration:
+That's the code for the UIKit, if `videoCall` is `true` you'll redner the `AgoraUIKit` component. If it is `false` we show a _Join_ button.
```
Comment (billy-the-fish) [inline shared/video-sdk/get-started/get-started-uikit/project-implementation/web.mdx:101]: render.  


if it is `false`, show the _Join_ button. 

(Never say we, avoid the person or say you. )

**Review by billy-the-fish — DISMISSED** [review]
Nice changes, thanks loads for the work. 
One small language change. Another one is up to you. With scrollycode, when the user clicks copy, they copy all that is in the code block, not just the code that is highlighted. With the current code, copy and paste coding won't work. Have a look in https://github.com/AgoraIO/Docs-Source/pull/314 and you can see how it works. I've asked Dennis to update so it works for all platforms. However, if you want to have a play and fix it, all good.

**Review by EkaanshArora — COMMENTED** [review]
I'll update the scrolly code in a separate PR, can you please approve the changes and I can merge.

**Review by billy-the-fish — APPROVED** [review]

**Comment by EkaanshArora** [comment]
Thanks!

**Pull request merged**